packages/mask/src/social-network-adaptor/mirror.xyz/collecting/identity.ts

```diff
@@ -25,20 +23,23 @@ function resolveLastRecognizedIdentityInner(
         const writer = await getCurrentUserInfo()
         if (!writer) return
 
-        ref.value = {
-            avatar: writer.avatarURL,
-            nickname: writer.displayName,
-            identifier: ProfileIdentifier.of(mirrorBase.networkIdentifier, writer.address).unwrapOr(undefined),
-            bio: writer.description,
-            homepage: writer.domain || getMirrorProfileUrl(writer.address),
-        }
+        ref.value = formatWriter(writer)
     }
 
     assign()
 
     window.addEventListener('locationchange', assign, { signal: cancel })
 }
 
+export const getUserInfo = () => {
```

Comment:
Move this API to `@masknet/web3-providers`.

Reply:
This method is get info from local, I renamed this method to `getUserInfoFromLocal`